Makefile

```diff
@@ -221,7 +221,8 @@ e2e-framework: ## Builds the CAPI e2e framework
 
 .PHONY: lint
 lint: $(GOLANGCI_LINT) ## Lint codebase
-	$(GOLANGCI_LINT) run -v $(GOLANGCI_LINT_EXTRA_ARGS)
+	# $(GOLANGCI_LINT) run -v $(GOLANGCI_LINT_EXTRA_ARGS)
```

Comment:
Looks like you left this commented out for core from testing?

Reply:
![image](https://user-images.githubusercontent.com/3118335/121389790-9a8cc880-c901-11eb-8abe-42c448fe41a7.png)